lib/Controller/InternalController.php

```diff
@@ -21,18 +21,16 @@
 
 class InternalController extends ApiController {
 	private $userId;
-	private $credentialService;
-	private $config;
 	private $manager;
```

Comment:
is there a reason leaving $manager, as it is?

Reply:
I wasn't sure whether it breaks the DI or not,  I've addressed it in the next commit